From 945436515ea11899138d4e72db1e0bdb3064401a Mon Sep 17 00:00:00 2001
From: Shadi Ammouri <shadi@marvell.com>
Date: Tue, 8 Mar 2016 15:23:24 +0200
Subject: [PATCH 248/538] xor: cp110: Add pcie/dma macro clock for DMA driver
 clock gating

DMA unit needs to enable the pcie-dma macro in addition to the DMA clock
for a proper functionality.

Change-Id: Ic8d654fcc82106676458e0094c9a9065cfcca3f3
Signed-off-by: Shadi Ammouri <shadi@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/28119
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-cp110.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
index c2ea45a..8f8ec10 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
@@ -121,7 +121,7 @@ cpxor0@6a0000 {
 	      <0x6b0000 0x1000>;
 	dma-coherent;
 	msi-parent = <&gic_v2m0>;
-	clocks = <&gateclk 8>;
+	clocks = <&gateclk 14>, <&gateclk 8>;
 };
 
 cpxor1@6c0000 {
@@ -130,7 +130,7 @@ cpxor1@6c0000 {
 	      <0x6d0000 0x1000>;
 	dma-coherent;
 	msi-parent = <&gic_v2m0>;
-	clocks = <&gateclk 7>;
+	clocks = <&gateclk 14>, <&gateclk 7>;
 };
 
 mdio@12a200 {
-- 
1.9.1

